service/controller/machinepool/handler/migration/testdata/new_machinepool_fully_created.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Test input, new MachinePool CR that is fully created (Metadata, Spec and Status are updated).